cmd/containerd/command/main.go

```diff
@@ -201,6 +202,7 @@ can be used and modified as necessary as a custom configuration.`
 			serve(ctx, l, server.ServeDebug)
 		}
 		if config.Metrics.Address != "" {
+			metrics.Register()
```

Comment:
Does the metrics package have Register method? The build is green though.

https://github.com/containerd/containerd/tree/master/metrics